Manifest.toml.gpuarrays-6

```diff
@@ -1,533 +0,0 @@
-# This file is machine-generated - editing it directly is not advised
```

Comment:
If we're going to delete these (which I'm fine with), we should remove the corresponding jobs in buildkite CI.

Reply:
Maybe have a separate PR for this?
I'd put moving host signal helpers to `Runtime` module in there as well...